plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/EdgeLabelsMoveFromEdgeMoveTest.java

```diff
@@ -697,33 +697,36 @@ public void testLabelStabilityWhenMovingPointOnObliqueEdge() {
         String diagramName = "EdgeWithObliqueSegments";
         Map<String, Dimension> edgeLabelExpectedPosition = new LinkedHashMap<>();
 
-        // Case Oblique A of the spec - The reference point is on a segment that
-        // is not impacted by the move --> The label does not move.
-        edgeLabelExpectedPosition.clear();
-        edgeLabelExpectedPosition.put("refToBEnd", new Dimension(0, 0));
-        doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, 50)), edgeLabelExpectedPosition, ZoomLevel.ZOOM_100, "A", "B", 1, true);
+        // // Case Oblique A of the spec - The reference point is on a segment that
+        // // is not impacted by the move --> The label does not move.
+        // edgeLabelExpectedPosition.clear();
+        // edgeLabelExpectedPosition.put("refToBEnd", new Dimension(0, 0));
+        // doTestMovePointOnEdge(diagramName, Arrays.asList(new Point(0, 50)), edgeLabelExpectedPosition,
```

Comment:
Why comment instead of simply delete the code?

Reply:
It was a mistake. I have not removed the temporary comments to debug the problem.